content/intro-to-storybook/vue/es/composite-component.md

```diff
@@ -0,0 +1,270 @@
+---
+title: 'Ensamblar un componente compuesto'
+tocTitle: 'Componente Compuesto'
+description: 'Ensamblar un componente compuesto a partir de componentes simples'
+commit: c72f06f
+---
+
+En el último capítulo construimos nuestro primer componente; este capítulo extiende lo que aprendimos para construir TaskList, una lista de Tasks (o Tareas). Combinemos componentes en conjunto y veamos qué sucede cuando se añade más complejidad.
+
+## Lista de Tareas
+
+Taskbox enfatiza las tareas ancladas colocándolas por encima de las tareas predeterminadas. Esto produce dos variaciones de `TaskList` para las que necesita crear historias: ítems por defecto e ítems por defecto y anclados.
+
+![default and pinned tasks](/intro-to-storybook/tasklist-states-1.png)
+
+Dado que los datos de `Task` pueden enviarse asincrónicamente, **también** necesitamos un estado de cargando para renderizar en ausencia de alguna conexión. Además, también se requiere un estado vacío para cuando no hay tareas.
+
+![empty and loading tasks](/intro-to-storybook/tasklist-states-2.png)
+
+## Empezar la configuración
+
+Un componente compuesto no es muy diferente de los componentes básicos que contiene. Crea un componente `TaskList` y un archivo de historia que lo acompañe: `src/components/TaskList.vue` y `src/components/TaskList.stories.js`.
+
+Comienza con una implementación aproximada de la `TaskList`. Necesitarás importar el componente `Task` del capítulo anterior y pasarle los atributos y acciones como entrada.
+
+```html
+<template>
+  <div>
+    <div class="list-items" v-if="loading">loading</div>
+    <div class="list-items" v-if="noTasks && !this.loading">empty</div>
+    <div class="list-items" v-if="showTasks">
+      <task
+        v-for="(task, index) in tasks"
+        :key="index"
+        :task="task"
+        @archiveTask="$emit('archiveTask', $event)"
+        @pinTask="$emit('pinTask', $event)"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+  import Task from './Task';
+  export default {
+    name: 'task-list',
+    props: {
+      loading: {
+        type: Boolean,
+        default: false,
+      },
+      tasks: {
+        type: Array,
+        default: () => [],
+      },
+    },
+    components: {
+      Task,
+    },
+    computed: {
+      noTasks() {
+        return this.tasks.length === 0;
+      },
+      showTasks() {
+        return !this.loading && !this.noTasks;
+      },
+    },
+  };
+</script>
+```
+
+A continuación, crea los estados de prueba de `Tasklist` en el archivo de historia.
+
+```javascript
```

Comment:
@IvanSotelo this snippet is incorrect, when i updated the vue documentation to use csf i copied over the wrong snippet, as soon as the pr that @tmeasday is working on is merged, i'll update the documentation accordingly, as this was a oversight on my part. i've changed it in the portuguese translation pr #257